File: python/hello.py
#!/usr/bin/python
import cgi, cgitb
import time
import random

cgitb.enable()
print 'Content-Type: text/html;charset=utf-8'
print
print '<!DOCTYPE html>'
print '<html>'

print '<head><title>Hello World from Python</title></head>'
r = random.randint(1, 16)
color_list = ['aqua', 'black', 'blue', 'fuchsia', 'gray', 'green', 'lime', 'maroon',
              'navy', 'olive', 'purple', 'red', 'silver', 'teal', 'white', 'yellow']
print '<body style="background-color:' + color_list[r-1] + ';">'
print '<h2>Hello World from Python@' + str(time.time()) + '</h2>'
print '</body></html>'

File: ruby/hello.rb
#!/usr/bin/ruby

require 'time'


listOfColors = ["aqua", "black", "blue", "fuchsia", 
	"gray", "green", "lime", "maroon", "navy", 
	"olive", "purple", "red", "silver", "teal", "white", "yellow"]


def getRandomColor(arr)
    ranNum = rand(arr.length) 
    return arr[ranNum]	
end


puts "Content-type:text/html\r\n\r\n"
puts '<html>'
puts '<head>'
puts '</head>'
puts "<body style = 'background-color:#{getRandomColor(listOfColors)}'>"
puts "<p>Hello Word from ruby@#{Time.new}</p>"
puts '</body>'
puts '</html>'


